src/backend/api/reservations.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const { max } = require("../database");
+const router = express.Router();
+const knex = require("../database");
+//Reservation
+//	Returns all reservations
+router.get("/", async (request, response) => {
+    let myTable = await knex ("Reservation");
+    response.json(myTable);
+})
+//add new reservation
+router.post("/", async (request, response) => {
+    const newReservation = request.body;
+    const insertedReservation = await knex("Meal").insert({
```

Comment:
Shouldn't this go into the `Reservation` table, and later when querying you'll join the two tables (either in SQL or in JavaScript)?